### PR TITLE
Adding Nikola (https://getnikola.com)

### PR DESCRIPTION
**Pull request by ralsina**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/nikola/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (isuruf) [inline recipes/nikola/meta.yaml:27]: You need to add all the requirements here,
https://github.com/getnikola/nikola/blob/master/requirements.txt
Conda names of these packages are usually lowercase names.

Comment (ralsina) [reply recipes/nikola/meta.yaml:27]: Ugh. Ok.

Comment (ralsina) [reply recipes/nikola/meta.yaml:27]: Thanks for the hint.

How can one test this, other than changing it blindly and waiting for CI? Any links? I see nothing here: https://conda-forge.org/#add_recipe

Comment (isuruf) [reply recipes/nikola/meta.yaml:27]: If you like docker, you can run `.circleci/run_docker_build.sh` script.

Comment (ralsina) [reply recipes/nikola/meta.yaml:27]: Nevermind, there are too many dependencies that are not packaged for conda, not going to do this.

**Comment by astrojuanlu** [comment]
> Nevermind, there are too many dependencies that are not packaged for conda, not going to do this.

Only PyRSS2Gen, as far as I could see. Not the end of the world, and some of us could have helped.

**Comment by ralsina** [comment]
On Tue, Apr 14, 2020 at 8:11 AM Juan Luis Cano Rodríguez <
notifications@github.com> wrote:

> Nevermind, there are too many dependencies that are not packaged for
> conda, not going to do this.
>
> Only PyRSS2Gen, as far as I could see. Not the end of the world, and some
> of us could have helped.
>
I was just taking a shot at doing this because there was an open issue in
our bug tracker, but I am clearly not the right person to tackle it.
Of course anyone is welcome if they want to try, and I will be happy to
help with any changes on the Nikola side of things.

OTOH, it's trivial to just "pip install nikola" inside a conda environment
and I don't see any difference in the outcome.

**Pull request closed without merging**